pages/get-involved.tsx

```diff
@@ -3,20 +3,115 @@ import {
   Button,
   Container,
   Grid,
+  List,
   Typography,
 } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
 import React from 'react';
 
 import IndividualPageHead from '../components/helper/IndividualPageHead.tsx';
 import ShareButtons from '../components/helper/ShareButtons.tsx';
 import AccordionBuilder from '../components/layout/AccordionBuilder.tsx';
+import BenefitListItem from '../components/layout/BenefitListItem.tsx';
 import FactCard from '../components/layout/FactCard.tsx';
 import HeroBanner from '../components/layout/HeroBanner.tsx';
 import ImageContainer from '../components/layout/ImageContainer.tsx';
 import PaperSection from '../components/layout/PaperSection.tsx';
 import SectionContainer from '../components/layout/SectionContainer.tsx';
 import content from '../content/get-involved.ts';
 
+function BenefitsOfJoiningUs() {
+  const theme = useTheme();
+  const matchesXS = useMediaQuery(theme.breakpoints.down('sm'));
+  const matchesSM = useMediaQuery(theme.breakpoints.down('md'));
+
+  let bottomImageAdjust;
+  if (matchesSM) {
+    bottomImageAdjust = '9.8rem';
+  } else {
+    bottomImageAdjust = '6.6rem';
+  }
```

Comment:
I got the idea behind this but I think we can solve this in a simple way by using some MUI things